languages/rust/exercises/concept/vec/.meta/example.rs

```diff
@@ -0,0 +1,32 @@
+/// return a list of 5 languages to learn
+pub fn languages() -> Vec<String> {
+    vec![
+        "Rust".to_string(),
+        "JavaScript".to_string(),
+        "Elixir".to_string(),
+        "Lua".to_string(),
+        "Scheme".to_string(),
+    ]
+}
+/// return a list of languages labeled by their order
+pub fn languages_ranked() -> Vec<String> {
+    languages()
+        .iter()
+        .enumerate()
+        .map(|(index, lang)| format!("{}: {}", index + 1, lang))
+        .collect()
+}
```

Comment:
This function requires the `iterators` concept, which should probably come after `vectors`.

Reply:
Great points! I'm going to rewrite the solutions using for loops. And maybe tweak the problem to not require the iterator solutions.